bestie/server/test_result.go

```diff
@@ -160,7 +160,9 @@ func handleTestResultEvent(bazelBuildEvent bes.BuildEvent, streamId *build.Strea
 		sbuf.WriteString(fmt.Sprintf("\tinvocationId: %s\n", stream.invocationId))
 		sbuf.WriteString(fmt.Sprintf("\tinvocationSha: %s\n", stream.invocationSha))
 		glog.Info(sbuf.String())
-	}
+  } else {
+    glog.Info(fmt.Sprintf("Processing invocationId: %s\n", stream.invocationId));
+  }
```

Comment:
Indentation should match original closing brace.

Reply:
Done.  gofmt's use of tabs instead of spaces trips me up every time.